.github/workflows/ci.yml

```diff
@@ -409,7 +409,7 @@ jobs:
       - uses: actions/setup-python@v4
         id: setup-python
         with:
-          python-version: 3.11.0-beta.4
+          python-version: "3.11"
```

Comment:
https://github.com/PyO3/pyo3/blob/6766d9f93bd2f4130953627d7dc6e8758acc8e16/emscripten/Makefile#L6

I think this also needs to be updated.

Reply:
Good catch! Removed the variable outright.